### ISSUE TITLE
425387DC - Wet Demo

### ISSUE THREAD

**Issue by voltagex** [xenia-project/game-compatibility#87]
benvanik/xenia@fc7695f
http://marketplace.xbox.com/en-US/Product/WET-Demo/00000000-0000-400d-80df-0001425387dc

Loads from disk (as in there's a reasonable amount of disk activity), then hangs.

```
>	xenia.exe!xe::apu::XmaDecoder::WorkerThreadMain() Line 118	C++
 	xenia.exe!xe::apu::XmaDecoder::Setup::__l2::<lambda>() Line 104	C++
 	[External Code]	
 	xenia.exe!xe::kernel::XHostThread::Execute() Line 769	C++
 	xenia.exe!xe::kernel::XThreadStartCallbackWin32(void * param) Line 365	C++
```

**Comment by XxTheK3LLERWithinxX84** [comment]
I tried the full game, same thing here... I have uploaded a log for this game.
[xenia log - wet.txt](https://github.com/xenia-project/game-compatibility/files/187771/xenia.log.-.wet.txt)

**Comment by voltagex** [comment]
Ooh, looks like I missed uploading a proper log for this one. Thanks for the test with benvanik/xenia@ea7bad10

**Comment by XxTheK3LLERWithinxX84** [comment]
I've used Xenia v1.0.285 to test this game out, it was made on 22 of this month.
and don't worry :D I tried with the older version but same problem.

**Comment by voltagex** [comment]
Yep, I grabbed the version number from the log. Mainly this is a note that
I don't need to re-test this for a while when I start again.

**Comment by XxTheK3LLERWithinxX84** [comment]
On 1.0.292, this problem still happens. Just a blank screen. The log file seems to stop at 46KB with this game...

**Comment by Gliniak** [comment]
Tested on: https://github.com/xenia-project/xenia/commit/b12554c5af3e8560b16385dee2e3eac08cce826f

### **Description:**
Freezes in main menu after 2 seconds. No input response in menu. Every thread stuck in limbo of waiting and one is suspended

**Log (level 3):** [xenia.log](https://github.com/xenia-project/game-compatibility/files/4283573/xenia.log)


![image](https://user-images.githubusercontent.com/153369/75818840-e6190400-5d99-11ea-8c40-f8cb6d8627fa.png)

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/fbc31652e0e31b94da547195623fd53252d513f6

Started a new game. Decided it was safer not to.

⁉️Xenia has huge memory leaks during scripted cut scenes!

![wet (4)](https://user-images.githubusercontent.com/31154606/120265321-619b8680-c265-11eb-9593-bdcc1e9174f7.png)


<details>
  <summary>More Screenshots</summary>

![wet (1)](https://user-images.githubusercontent.com/31154606/120265323-63fde080-c265-11eb-8c27-b4d49ffd6da9.png)
![wet (2)](https://user-images.githubusercontent.com/31154606/120265324-64967700-c265-11eb-9bf9-22ecaa35c657.png)
![wet (3)](https://user-images.githubusercontent.com/31154606/120265325-64967700-c265-11eb-9f70-3457ea37eb23.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/6573291/xenia.zip)

state-gameplay | tech-middleware-havok

**Comment by Etokapa** [comment]
Tested on https://github.com/xenia-project/xenia/commit/4498a28568b629d7929cfdfbc35ff374e0a0e3ab

Completed entire demo. Xenia no longer rekts your computer while playing. 40-100FPS with vsync off and shaders cached.

![wet (2)](https://user-images.githubusercontent.com/31154606/124002492-6c982280-d99b-11eb-939d-45d581173866.png)

<details>
  <summary>More Screenshots</summary>

![wet (1)](https://user-images.githubusercontent.com/31154606/124002507-70c44000-d99b-11eb-827b-3249945eaddb.png)
![wet (4)](https://user-images.githubusercontent.com/31154606/124002510-715cd680-d99b-11eb-8da5-11e7fa3cd343.png)
![wet (5)](https://user-images.githubusercontent.com/31154606/124002511-71f56d00-d99b-11eb-9386-a25e9091febd.png)

</details>

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/6742736/xenia.zip)
[xenia.z01.zip](https://github.com/xenia-project/game-compatibility/files/6742737/xenia.z01.zip)
[xenia.z02.zip](https://github.com/xenia-project/game-compatibility/files/6742734/xenia.z02.zip)

state-playable
